frontend/src/router.js

```diff
@@ -3,22 +3,22 @@ import VueRouter from "vue-router";
 
 Vue.use(VueRouter);
 
-const HelloWorld = () => import("./pages/HelloWorld.vue");
-const Screening = () => import("./pages/Screening.vue");
-const Matching = () => import("./pages/Matching.vue");
-const StudentProfile = () => import("./pages/StudentProfile.vue");
-const NewStudent = () => import("./pages/NewStudent.vue");
+const HelloWorld = () => import('./pages/HelloWorld.vue')
+const Screening = () => import('./pages/Screening.vue')
+const Matching = () => import('./pages/Matching.vue')
+const NewStudent = () => import('./pages/NewStudent.vue')
+const AllStudents = () => import('./pages/Students.vue')
 
 const router = new VueRouter({
   mode: "history",
   hash: false,
   routes: [
-    { path: "/", component: HelloWorld },
-    { path: "/screening", component: Screening },
-    { path: "/matching", component: Matching },
-    { path: "/student/:id", component: StudentProfile },
-    { path: "/new", component: NewStudent },
-  ],
-});
+    { path: '/', component: HelloWorld  },
```

Comment:
Are you knowingly using Prettier? It should not be changing all the double quotes to single quotes, since we have a `.prettierrc` file. But this is not a big deal for now.